src/results/financial.jl

```diff
@@ -146,7 +146,8 @@ function initial_capex(m::JuMP.AbstractModel, p::REoptInputs; _n="")
     for b in p.s.storage.types.elec
         if p.s.storage.attr[b].max_kw > 0
             initial_capex += p.s.storage.attr[b].installed_cost_per_kw * value.(m[Symbol("dvStoragePower"*_n)])[b] + 
-                p.s.storage.attr[b].installed_cost_per_kwh * value.(m[Symbol("dvStorageEnergy"*_n)])[b]
+                p.s.storage.attr[b].installed_cost_per_kwh * value.(m[Symbol("dvStorageEnergy"*_n)])[b] +
+                p.s.storage.attr[b].installed_cost_constant
```

Comment:
will this report a capital cost if the recommended storage size is zero?

Reply:
I think it would. I added a multiplier to the installed_cost_constant on line 150. The multiplier should ensure that the installed_cost_constant is only included if the "m[Symbol("dvStorageEnergy"*_n)]" variable is greater than 0.